Worktez/worktez
# fix(check username button): the check username button reusable in the edit profile section

### Functionality:
 the check username button reusable in the edit profile section

### Solution:
Changes in  edit-profile component

### Risk level:

### How to test:
in profile page click on edit profile button then change username and click on check availability button
